logging/hooks/custom_logs.go

```diff
@@ -0,0 +1,57 @@
+package hooks
+
+import (
+	"github.com/avenga/couper/config/request"
+	"github.com/avenga/couper/eval"
+	"github.com/hashicorp/hcl/v2"
+	"github.com/sirupsen/logrus"
+)
+
+var _ logrus.Hook = &CustomLogs{}
+
+const customLogField = "custom"
+
+type CustomLogs struct{}
+
+func (c *CustomLogs) Levels() []logrus.Level {
+	return logrus.AllLevels
+}
+
+func (c *CustomLogs) Fire(entry *logrus.Entry) error {
+	if entry.Context != nil {
+		if t, exists := entry.Data["type"]; exists {
+			switch t {
```

Comment:
Both `type`'s are configurable. We have to pass the logging configuration to this hook.

Reply:
Done